File: Frontend/src/services/flight-models/flight.js
export default class Flight {
     id;
     fromAirportId;
     toAirportId;
     departureTime;
     arrivalTime;
     airplaneId;
     suitcaseMassKg;
     suitcaseCount;
     handLuggageMassKg;
     handLuggageCount;
     massOverloadKgCost;

     constructor (id, fromAirportId, toAirportId, departureTime, arrivalTime, airplaneId, suitcaseMassKg, suitcaseCount, handLuggageMassKg, handLuggageCount, massOverloadKgCost) {
         this.id = id;
         this.fromAirportId = fromAirportId;
         this.toAirportId = toAirportId;
         this.departureTime = departureTime;
         this.arrivalTime = arrivalTime;
         this.airplaneId = airplaneId;
         this.suitcaseMassKg = suitcaseMassKg;
         this.suitcaseCount = suitcaseCount;
         this.carryonMassKg = handLuggageMassKg;
         this.handLuggageCount = handLuggageCount;
         this.massOverloadKgCost = massOverloadKgCost;
     }
}

File: Server/WebAPI/Models/Flight.cs
using System;

namespace WebAPI.Models
{
    public class Flight
    {
        public int? Id { get; set; }
        public int FromAirportId { get; set; }
        public int ToAirportId { get; set; }
        public DateTimeOffset DepartureTime { get; set; }
        public DateTimeOffset ArrivalTime { get; set; }
        public int AirplaneId { get; set; }
        public int SuitcaseMassKg { get; set; }
        public int SuitcaseCount { get; set; }
        public int HandLuggageMassKg { get; set; }
        public int HandLuggageCount { get; set; }
        public int MassOverloadKgCost { get; set; }
    }
}